sphinx/cmd/quickstart.py

```diff
@@ -245,60 +230,63 @@ def ask_user(d):
 Please enter values for the following settings (just press Enter to
 accept a default value, if one is given in brackets).''')
 
-    if 'path' in d:
+    if d.get('path'):
```

Comment:
I prefer to use `in` and `not in` operator to check an item exists or not. I feel it's natural.

Reply:
These do totally different things. Previously, we had these lines inside in `main`:

    # delete None or False value		
    d = dict((k, v) for k, v in d.items() if not (v is None or v is False))

This would mean these keys would not be present. This is no longer the case because of how we're storing defaults. As such, we do not need to check if the keys are present but rather what their value is. Compare the two things below:

    >>> x = {'path': None}
    >>> if 'path' in x:
    ...     print('Got path (A)!')
    ... 
    Got path (A)!
    >>> if x.get('path'):
    ...     print('Got path (B)!')
    ... 

Note that we use `d.get(key)` here to simplify our unit tests. If we didn't do this and used `d[key]` notation, we'd have to ensure the value of `d` contained all keys here.